assets/dev/js/editor/regions/navigator/navigator.js

```diff
@@ -11,6 +11,7 @@ export default class extends BaseRegion {
 		this.component = $e.components.register( new Component( { manager: this } ) );
 
 		this.isDocked = false;
+		this.storage.size.width = this.$el.css( 'width' );
```

Comment:
What happens if the navigator is hidden on page load?

Reply:
It works as intended because `JQuery.css()` uses `getComputedStyle()` which retrieves the width from the css rules.